src/split-panel/__tests__/utils.test.tsx

```diff
@@ -17,48 +16,67 @@ describe('use-keyboard-events', () => {
 
   test('bottom position, up key', () => {
     const onKeyDown = useKeyboardEvents(sizeControlProps);
-    const e = new KeyboardEvent('keydown', { keyCode: KeyCode.up });
-    onKeyDown(e as any);
+    const event = new KeyboardEvent('keydown', { keyCode: KeyCode.up });
+    onKeyDown(event as any);
 
-    expect(sizeControlProps.setBottomPanelHeight).toHaveBeenCalledTimes(1);
-    expect(sizeControlProps.setBottomPanelHeight).toHaveBeenCalledWith(110);
+    expect(sizeControlProps.onResize).toHaveBeenCalledTimes(1);
+    expect(sizeControlProps.onResize).toHaveBeenCalledWith(110);
   });
 
   test('side position, right key', () => {
     const onKeyDown = useKeyboardEvents({ ...sizeControlProps, position: 'side' });
-    const e = new KeyboardEvent('keydown', { keyCode: KeyCode.right });
-    onKeyDown(e as any);
+    const event = new KeyboardEvent('keydown', { keyCode: KeyCode.right });
+    onKeyDown(event as any);
 
-    expect(sizeControlProps.setSidePanelWidth).toHaveBeenCalledTimes(1);
-    expect(sizeControlProps.setSidePanelWidth).toHaveBeenCalledWith(90);
+    expect(sizeControlProps.onResize).toHaveBeenCalledTimes(1);
+    expect(sizeControlProps.onResize).toHaveBeenCalledWith(90);
+  });
+
+  test('bottom position, pageUp key', () => {
```

Comment:
This is not really a new feature in my PR, but codecov complained about missed coverage and required me to fix this to pass the workflow